### PR TITLE
Add Centurion

### PR DESCRIPTION
**Pull request by v1993**
Add https://github.com/albin-johansson/centurion - a C++ header-only wrapper for SDL2.

### CODE REVIEW

File: subprojects/packagefiles/centurion/meson.build [hunk 1]
```diff
@@ -0,0 +1,40 @@
+project('centurion', 'cpp', version: '7.2.0', license: 'MIT', default_options: ['cpp_std=c++17'])
+
+centurion_args = []
+centurion_deps = [dependency('sdl2')]
+
+# Uppercase is used by pkg-config, (built-in) cmake finders and warps are lowercase
```
Comment (eli-schwartz) [inline subprojects/packagefiles/centurion/meson.build:6]: Typo: warps

File: subprojects/packagefiles/centurion/meson.build [hunk 2]
```diff
@@ -0,0 +1,40 @@
+project('centurion', 'cpp', version: '7.2.0', license: 'MIT', default_options: ['cpp_std=c++17'])
```
Comment (eli-schwartz) [inline subprojects/packagefiles/centurion/meson.build:1]: This should specify the lowest meson_version it supports.

**Review by eli-schwartz — COMMENTED** [review]
Seems like upstream doesn't even provide a dependency finder config file? :eyes:

File: subprojects/gvdb.wrap [hunk 3]
```diff
@@ -0,0 +1,2 @@
+[wrap-redirect]
+filename = glib-2.74.4/subprojects/gvdb.wrap
```
Comment (v1993) [inline subprojects/gvdb.wrap:2]: Whoops, this was unintentional (looks like a byproduct of sanity checker script).

File: subprojects/gvdb.wrap [hunk 4]
```diff
@@ -0,0 +1,2 @@
+[wrap-redirect]
```
Comment (eli-schwartz) [inline subprojects/gvdb.wrap:1]: Where did this file come from?

**Comment by v1993** [comment]
What do those CI errors mean? I'm unfamiliar with wrapdb's tooling, so they are rather cryptic to me...

File: releases.json [hunk 5]
```diff
@@ -125,6 +125,14 @@
       "1.1.5-1"
     ]
   },
+  "centurion": {
+    "dependency_names": [
+      "centurion"
+    ],
+    "versions": [
+      "7.2.0"
```
Comment (eli-schwartz) [inline releases.json:133]: This version needs to include both the upstream release number and the WrapDB local build number (starting with "1") separated by a hyphen.

See https://github.com/mesonbuild/wrapdb#how-to-contribute-new-wraps as well as the style used by other entries in this file.

Comment (v1993) [reply releases.json]: Ah, thank you, just figured it out!

**Comment by v1993** [comment]
Is there anything I can do to help with this getting merged?

**Pull request merged**